WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -208,6 +204,14 @@ - (void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withT
 
 #pragma mark - Data Model setup
 
+- (NSString *)adminRowTitle
+{
+    if (self.blog.isHostedAtWPcom) {
+        return NSLocalizedString(@"Dashboard", @"Action title. Noun. Opens the user's WordPress.com dashboard in an external browser.");
```

Comment:
The "Dashboard" is wp-admin. I haven't seen anyone referring to calypso as the "dashboard" yet. And specially because this would take users to stats, not a site dashboard.